public/css/dashboard-application.css

```diff
@@ -118,23 +97,82 @@
 	margin-bottom: 30px;
 }
 
+.mainHeader h1 {
+	text-align: center;
+	display: inline-block;
+	background-color: #efa516;
+	overflow-wrap: break-word;
+	width: 682px;
+	height: 76px;
+	padding: 15px 0px 15px 0px;
+	color: white;
+	font-family: "Roboto", sans-serif;
+	font-style: italic;
+	font-weight: bold;
+	font-size: 40px;
+	line-height: 47px;
+	text-shadow: 0px 4px 4px rgba(0, 0, 0, 0.25);
+	margin-bottom: 40px;
+}
+
+@media (max-width: 682px) {
```

Comment:
This should be max-width 768 to match with bootstrap

Reply:
Definitely should be changed to match Bootstrap, but instead to match the small breakpoint (576px) so the headers don't stretch too far out just after the breakpoint

Reply:
@robertnperalta I changed this to 720px, it seemed like when I matched the small breakpoint there was some issues with the text overflowing